content/english/guidelines/formatting.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Formatting"
+date: 2020-07-29T14:08:32-07:00
+draft: false
+hidden: true
+---
+
+## Formatting Reference
+
+- [Standard Markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+- [Custom Formatting Options using shortcodes](shortcodes/)
+
+## Formatting Examples
+
+This theme includes a number of options to customize the look and feel of your site.
+
+### Alerts
+
+#### Info
+
+    {{%/* alert theme="info" */%}}**this** is an info {{%/* /alert */%}}
+
+{{% alert theme="info"%}}**this** is an info{{% /alert %}}
+
+#### Success
+
+    {{%/* alert theme="success" */%}}**Yeahhh !** is a success{{%/* /alert */%}}
+
+{{% alert theme="success" %}}**Yeahhh !** is a success{{% /alert %}}
+
+#### Warning
+
+    {{%/* alert theme="warning" */%}}**Be carefull** is a warning{{%/* /alert */%}}
+
+{{% alert theme="warning" %}}**Be carefull** is a warning{{% /alert %}}
```

Comment:
```suggestion
```suggestion
    {{%/* alert theme="warning" */%}}**Be carefull** is a warning{{%/* /alert */%}}

{{% alert theme="warning" %}}**Be carefull** is a warning{{% /alert %}}
```
    {{%/* alert theme="warning" */%}}**Be carefull** is a warning{{%/* /alert */%}}

{{% alert theme="warning" %}}**Be carefull** is a warning{{% /alert %}}
```
```suggestion
    {{%/* alert theme="warning" */%}}**Be careful** is a warning{{%/* /alert */%}}

{{% alert theme="warning" %}}**Be careful** is a warning{{% /alert %}}
```